src/Uno.UI/UI/Xaml/Controls/ListViewBase/ListViewBase.cs

```diff
@@ -105,13 +105,21 @@ internal bool TryHandleKeyDown(KeyRoutedEventArgs args)
 					OnItemClicked(focusedContainer, args.KeyboardModifiers);
 				}
 			}
-			else if (args.Key == VirtualKey.Down)
-			{
-				return TryMoveKeyboardFocusAndSelection(+1, args.KeyboardModifiers);
-			}
-			else if (args.Key == VirtualKey.Up)
+			else
 			{
-				return TryMoveKeyboardFocusAndSelection(-1, args.KeyboardModifiers);
+				var orientation = ItemsPanelRoot.InternalOrientation ?? Orientation.Vertical;
```

Comment:
nit: InternalOrientation can be non-nullable with default implementation returning Vertical.